bitnami/zookeeper/README.md

```diff
@@ -41,6 +41,10 @@ $ helm delete my-release
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
+## Log level
+
+You can configure the Zookeeper log level using the `ZOO_LOG_LEVEL` environment variable. By default, it is set to `FALSE` because of each readiness prove produce an `INFO` message on connection and a `WARN` message on disconnection.
```

Comment:
This should probably say "set to `ERROR`"

Reply:
According to the README table some lines below:
```
| `logLevel`                            | Log level of Zookeeper server                                       | `ERROR`       |
```
it is set to `ERROR` by default

Reply:
Yes, you are right, it was a mistake.